### PR TITLE
add Notes for deprecated licenses

### PR DESCRIPTION
**Pull request by jlovejoy**
add note re: deprecation

see issue #1006 for more info

### CODE REVIEW

File: src/GPL-2.0-with-classpath-exception.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <crossRefs>
          <crossRef>https://www.gnu.org/software/classpath/license.html</crossRef>
       </crossRefs>
-
+     <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: Classpath-exception-2.0</notes>
```
Comment (waldyrious) [inline src/GPL-2.0-with-classpath-exception.xml:12]: Alignment is slightly off:
```suggestion
      <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: Classpath-exception-2.0</notes>
```

File: src/GPL-3.0-with-GCC-exception.xml [hunk 2]
```diff
@@ -9,6 +9,7 @@
       <crossRefs>
          <crossRef>https://www.gnu.org/licenses/gcc-exception-3.1.html</crossRef>
       </crossRefs>
+	   <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: GCC-exception-3.1</notes>
```
Comment (waldyrious) [inline src/GPL-3.0-with-GCC-exception.xml:12]: Alignment is using mixed tab and space characters.
```suggestion
      <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: GCC-exception-3.1</notes>
```

File: src/StandardML-NJ.xml [hunk 3]
```diff
@@ -8,6 +8,8 @@
       <crossRefs>
          <crossRef>http://www.smlnj.org//license.html</crossRef>
       </crossRefs>
+       <notes>DEPRECATED: Duplicate license, use identifier: SMLNJ
+      </notes>
```
Comment (waldyrious) [inline src/StandardML-NJ.xml:12]: To match the other entries, the closing tag should be in the same line:
```suggestion
       <notes>DEPRECATED: Duplicate license, use identifier: SMLNJ</notes>
```

File: src/wxWindows.xml [hunk 4]
```diff
@@ -7,6 +7,8 @@
       <crossRefs>
          <crossRef>https://opensource.org/licenses/WXwindows</crossRef>
       </crossRefs>
+      <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: WxWindows-exception-3.1
+      </notes>
```
Comment (waldyrious) [inline src/wxWindows.xml:11]: To match the other entries, the closing tag should be in the same line:
```suggestion
      <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: WxWindows-exception-3.1</notes>
```

**Comment by jlovejoy** [comment]
@swinslow - ready to merge if checks all pass, I think

File: src/GPL-2.0-with-bison-exception.xml [hunk 5]
```diff
@@ -9,6 +9,7 @@
       <crossRefs>
          <crossRef>http://git.savannah.gnu.org/cgit/bison.git/tree/data/yacc.c?id=193d7c7054ba7197b0789e14965b739162319b5e#n141</crossRef>
       </crossRefs>
+	   <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: Bison-exception-2.2</notes>
```
Comment (waldyrious) [inline src/GPL-2.0-with-bison-exception.xml:12]: One last tab vs. spaces issue:

```suggestion
      <notes>DEPRECATED: Use license expression including main license, "WITH" operator, and identifier: Bison-exception-2.2</notes>
```

**Review by swinslow — APPROVED** [review]

**Comment by swinslow** [comment]
LGTM, thanks @jlovejoy! Merging now.

**Pull request merged**